docs/source/devguide/dev_assisted_package_creation.rst

```diff
@@ -0,0 +1,104 @@
+ .. dropdown:: Distribution Statement
+
+    | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+    | # # #
+    | # # # Author:
+    | # # # Naval Research Laboratory, Marine Meteorology Division
+    | # # #
+    | # # # This program is free software: you can redistribute it and/or modify it under
+    | # # # the terms of the NRLMMD License included with this program. This program is
+    | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+    | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+    | # # # for more details. If you did not receive the license, for more information see:
+    | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Developer Assisted Package Creation
+===================================
+
+Why We Assist With Package Creation
+-----------------------------------
+
+The GeoIPS Development team is committed to integrating a variety of geospatial
+processing routines into GeoIPS, or as a standalone packages that can interact with
+GeoIPS. Specifically we developers at CIRA are tasked with integrating proprietary
+software developed by researchers to work alongside GeoIPS. If you have access, see the
+`GeoIPS GLM Package <https://bear.cira.colostate.edu/overcast/geoips_glm>`_ for an
+example of a standalone package that implements GREMLIN and GLM
+(Global Lightning Mapper) routines working alongside GeoIPS.
+
+The purpose of integrating software like this is to elevate GeoIPS as a package that can
+process nearly any type of geospatial data, and output it in a variety of formats. For
+each standalone package that we create, we only elevate the capabilities of GeoIPS as a
+whole. Eventually, we'd like to see GeoIPS be the only package you need to handle your
+geospatial data routines.
+
+Use Cases That We'll Support
+----------------------------
+
+If you are a user, developer, or researcher that has software related to geospatial data
+processing and are unable to integrate that functionality into GeoIPS, you've come to
+the right place. We first request that the data processing you've implemented doesn't
+already exist in GeoIPS, or any of its corresponding packages. For a full list of public
+repositories that NRLMMD-GEOIPS supports, see
+`GeoIPS Repositories <https://github.com/orgs/NRLMMD-GEOIPS/repositories>`_.
+
+If the data routines you've created don't match any of the repositories linked above,
+then we've likely found a use case worth creating a separate GeoIPS package for.
+Assuming you've found a developer able to help you to get your processing routines
```

Comment:
👁️ 👁️  - I thought this was saying that _we_ are the developers that will help?

Reply:
You're right. What should I replace it with here? A link to the issue page or do we want to create a separate forum for this kind of stuff?

Reply:
Replaced it with a link to the issue page and ``geoips@nrlmry.navy.mil`` contact email for now.

Reply:
Thanks evan! That's good for now, we can link to the contact page in the future :)